### PR TITLE
Filtering out . 

### PR DESCRIPTION
**Pull request by stephenrlouie**
### Change log

- Filters out `.` from Plugin & Service `Names`
- Filters out `.` from any Input & Output `Names`

Ensures splits in [manager](https://github.com/sigma-dev/sigma/blob/master/manager/manager.go#L400) are not thrown off by user input

### CODE REVIEW

File: api/v1/server/server.go [hunk 1]
```diff
@@ -357,3 +358,40 @@ func (s *Server) UpdateService(w http.ResponseWriter, r *http.Request) {
...
+			if strings.Contains(i.Name, ".") {
+				return true
+			}
+		}
+		for _, o := range s.Output {
+			if strings.Contains(o.Name, ".") {
+				return true
+			}
+		}
+	case apiclient.Plugin:
```
Comment (meganokeefe) [inline api/v1/server/server.go]: add a default error case to the switch?

File: api/v1/server/server_test.go [hunk 2]
```diff
@@ -292,6 +306,84 @@ var fullService = apiclient.Service{
...
+		apiclient.ServiceIo{
+			Name:  "in.1",
+			Type_: "in1",
+			Value: nil,
+			From:  nil,
+		},
+	},
+}
+
+var periodNamePlugin = apiclient.Plugin{
```
Comment (meganokeefe) [inline api/v1/server/server_test.go]: lolz

**Comment by meganokeefe** [comment]
Question, why are we kicking out plugins with periods in them? As opposed to doing a find and replace on the `Names` and just inserting the plugin anyway. 

But I see the merit of throwing an error, for clarity.

**Comment by stephenrlouie** [comment]
manager implies meaning based off a `split(str, ".")` [code](https://github.com/sigma-dev/sigma/blob/master/manager/manager.go#L407-L412)

File: api/v1/server/server.go (hunk 1 above)
Comment (stephenrlouie) [inline api/v1/server/server.go]: yea, ill just return false, which will trigger the `StatusBadRequest`

**Pull request merged**